docs/adrs/0011-nimbus-web-glean-application-structure.md

```diff
@@ -0,0 +1,122 @@
+All apps supported by a single Cirrus container, as best we can estimate at this time, will all be a part of the same product domain. 
+In the previous example, the product domain is Pocket.
+
+The problem this ADR addresses is as follows: with Cirrus, how should we structure Glean applications for cross-platform apps integrating Cirrus?
+
+> #### Note
+> Various explanations below will reference Pocket, but this ADR generally applies to all products planning to integrate Nimbus into their applications.  
+
+## Decision Drivers
+
+* [Probe-scraper repository list](https://github.com/mozilla/probe-scraper/blob/main/repositories.yaml) developer burden
+* Ease of data collation
+  * How easy is it to collate data for an application?
+  * How easy is it to collate data across applications?
+* Glean Python SDK architecture
+
+## Considered Options
+
+* One Glean app ID
+* One Glean app ID per domain
+* One glean app ID per application
+
+## Decision Outcome
+
+Chosen option: "One Glean app ID per product domain", because it has the most advantages and fewest disadvantages _(see below)_. 
+Option "One Glean app ID" is also viable, but it could come with additional disadvantages in the form of conflicting data retention policies across product domains.
+Option "One glean app ID per application"'s disadvantages are simply too detrimental to the project for the option to be viable.
+
+### Positive Consequences
+
+* Its usage will work with the existing Glean Python SDK architecture
+* Collating data for one app and across apps is straightforward
+* Probe-scraper repository list will have overall fewer listings for Nimbus web applications than the current model
+* Sidecar deployment model remains viable within Pocket's architecture
+* Data retention can be set differently for each product domain
+
+### Negative Consequences
+
+* Jetstream will require changes in order to support this choice
+* We are breaking away from our historical standard of Glean practices
+* We may have to make more intensive changes to Experimenter
+
+## Pros and Cons of the Options
+
+### One Glean app ID
+
+With this option, we would have one Glean application ID that is shared across all instances of Cirrus.
+All applications using Cirrus would share a single Glean application ID, allowing for an overall lower maintenance burden for probe-scraper and easier collation of data across applications.
+One downside of this solution is that data from multiple product domains will exist within the same table structure.
+
+* Good, because it has the smallest footprint in the probe-scraper repository of the listed options.
+* Good, because the Glean SDK supports this particular use case.
+* Good, because this model supports Pocket's architecture.
+* Bad, because data retention cannot be handled separately for each application or product domain.
+* Bad, because it breaks away from our existing Glean usage practices.
```

Comment:
I'm not sure what this means?  How does this impact using Glean in our project?

Reply:
This is referring to how Glean applications are configured at this time — one app id per application (fenix, firefox_ios, etc) and its use in client-side applications versus servers.

This has no impact as far as I can tell, just wanted to note that this option is a departure from the norm.